Makefile

```diff
@@ -124,6 +125,9 @@ bridge: | build deps
 # Builds and run the test suite (Waku v1 + v2)
 test: | test1 test2
 
+installganache: 
```

Comment:
As mentioned in call, feel free to make this optional at PoC stage with 1) a comment on problem 2) issue to fix it

Reply:
3) Comment in README on Ganache pre-req and how to run this test

Reply:
The following PR https://github.com/status-im/nim-waku/pull/349 automates the process of ganache installation and launch, as soon as it gets merged into the master branch,  there will be no need to install ganache manually